### ISSUE TITLE
Display google calendar not working

### ISSUE THREAD

**Issue by schuesslerf** [fullcalendar/fullcalendar#5716]
### Reduced Test Case

 not reproducible with a link

### Bug Description

I've been trying to integrate my google calendar inside of FullCalendar.io.

My code without the google integration looks like the following:

```
<div id="calendarElem">
<link href='fullcalendar/main.css' rel='stylesheet' />
<script src='fullcalendar/main.js'></script>
<script>
$(function(){
    var calendarEl = document.getElementById('calendarElem');
    var calendar = new FullCalendar.Calendar(calendarEl, {
        initialView: 'dayGridMonth'
    });
    calendar.render();
});
</script>
</div>
```

Using that code, I'm getting an empty calendar - so far so good.

Then I tried following the docs to integrate the google calendar.

I went and created an API key for Google Calendar, made my calendar public and copied the URL to it.

Then, using the following code, I got an error saying "googleCalendarPlugin is not defined":

```
var calendar = new FullCalendar.Calendar(calendarEl, {
        initialView: 'dayGridMonth',
        plugins: [ googleCalendarPlugin ],
        googleCalendarApiKey: 'aGeneratedKey',
        events: {
            googleCalendarId: 'myCalendarURL'
        }
    });
```
I then tried to simply remove the plugin from there, which solved the previous problem, however, now I get the result from Google, saying that my API key is not correct.... This might make sense, because I couldn't follow the exact procedure from Fullcalendar, as in the docs, it says to create a "browser key", however, Google only provided me with the possibility to create a OAuth client ID or a service account, not to create a browser key...

It seems like I then somehow managed to create an API key that works for the calendar, by simply creating an API key in general, not specific for the calendar API...It now doesn't show the error anymore, that the API-key is invalid.

However, now I'm getting a 404, I guess this means that the googleCalnedarID is not correct...might be the case, because it looks completely different from what the docs show as well.. I've tried the complete link when clicking on the share button, I've tried only the last part of that which seemed to be an ID, I even tried the ID from the URL when trying to share it... Problem is that everything is not like described and hard to get it working...

**Comment by acerix** [comment]
It looks like the process changed quite a bit since those docs, seems "browser key" is called "API key" now.  

I was able to create a key under Credentials > Create Credentials > API Key, and it works fine in the holiday demo:

https://codepen.io/acerix/pen/wvGMBXr?editors=001

The instuctions to share the calendar and get the calendar ID worked for me, and it works when I put the ID it in the demo above.  Although the calendar ID for my Google account is actually my email address, not something like abcd1234@group.calendar.google.com as in the docs.

**Comment by schuesslerf** [comment]
@acerix Yes, I got that creating the API key part, seems like that worked... Considering the calendar ID, how did you get it? did you just click on the  "copy link" button and paste the hole link in? tried everything possible there....

**Comment by acerix** [comment]
I just copy/pasted the text under Calendar ID (which is my email address). Easy to miss though, I found it with this screenshot:

https://help.crystalcommerce.com/en/articles/1389811-how-to-find-your-google-calendar-id

**Comment by schuesslerf** [comment]
@acerix thank you so much!! saved my day! was just my email adress as well.. got it working now! although docs should be updated :)
